typings/sbvr-parser.d.ts

```diff
@@ -1,10 +1,4 @@
 declare module '@resin/sbvr-parser' {
-	import {
```

Comment:
This was never needed and I presume was from copy/pasting from another .d.ts file